app/helpers/lookup_keys_helper.rb

```diff
@@ -75,7 +75,8 @@ def lookup_key_with_diagnostic(obj, lookup_key, lookup_value)
       :name => "#{lookup_value_name_prefix(lookup_key.id)}[value]",
       :disabled => !lookup_key.overridden?(obj) || lookup_value.use_puppet_default,
       :inherited_value => inherited_value,
-      :lookup_key => lookup_key)
+      :lookup_key => lookup_key,
+      :lookup_key_type => lookup_key.key_type)
```

Comment:
Yes, i thought that should have worked! But the tests failed with undefined method error - http://ci.theforeman.org/job/test_develop_pr_core/database=postgresql,label=fast,ruby=2.1/8187/

I'll try and work it out, Thanks @dLobatog!